CONTRIBUTING.md

```diff
@@ -201,6 +201,29 @@ and would get rendered like
 
 #### List of shortcodes with explanation
 
+- `platform_support_table`: A table that displays information regarding which
+providers support a feature described in the context. Look for examples in the
+code base too understand how it is configured. The shortcode offers three
+parameters `aws`, `azure`, and `kvm`. Each one functions the same way:
```

Comment:
```suggestion
parameters `aws`, `azure`, `kvm` and `roadmap`. Each one functions the same way:
```

Reply:
Nope, there is no `roadmap` parameter. `roadmap` can be used as a key inside each of these parameters.